servers.json

```diff
@@ -1,4 +1,10 @@
-[ 
+[   {
+        "id": "shadowrient",
+        "name": "Shadowrient Network",
+        "addresses": [
+            "mc.shadowrient.ml"
```

Comment:
Remove the `mc.` prefix from this domain.

Reply:
changed!